packages/mantine/src/theme/Theme.tsx

```diff
@@ -134,6 +151,15 @@ export const plasmaTheme: MantineThemeOverride = {
                 },
             },
         },
+        DateInput: {
+            styles: {
+                root: {
+                    dropdown: {
+                        display: 'none',
+                    },
+                },
+            },
+        },
```

Comment:
I saw that we added that already in our table date pickers, do we need to add it in the theme as well?

Reply:
I've added it for testing but forgot to remove it afterward. Consider it removed